src/userbase-js/src/db.js

```diff
@@ -1055,7 +1055,13 @@ const _readBlob = async (blob) => {
   })
 }
 
-const _uploadChunk = async (batch, chunk, dbId, fileId, fileEncryptionKey, chunkNumber) => {
+const _uploadChunkRequest = async (request, bytesTransferredObject, progressHandler) => {
+  await request
+  bytesTransferredObject.bytesTransferred += FILE_CHUNK_SIZE;
```

Comment:
Small thing: I'd recommend passing `chunk.size` into `_uploadChunkRequest`, then using that here instead of `FILE_CHUNK_SIZE`. Otherwise last chunk would push the total bytesTransferred over what the caller would expect

Reply:
Alright, updated it!